alertproduction.tex

```diff
@@ -28,7 +28,8 @@ \section{Alert Production in Commissioning and Early Operations}
 Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following System First Light  (\S~\ref{sec:timeline}).
 \citeds{RTN-061} describes the criteria for sending the first Rubin alerts.
 Once begun, Alert Production will then proceed continuously into the full LSST survey.
-Alerts generated during commissioning may be produced with higher latency, and access to images and the PPDB may not be immediately available.
+Alerts generated during the early science period may be produced with higher latency, and access to images and the PPDB may not be immediately available.
+Alert packet contents for moving objects might not include the associated historical source records during the first phases of Alert Production, and parameters such as phase curve slope (G) and orbital elements would be empty until sufficient detections exist to derive them.
```

Comment:
Propose to strike "and orbital elements" as we'll always have these (they're a prerequisite for the object to me associated).

Reply:
This was resolved by committing Eric's suggested text thanks Mario.